app/queue/export_service.py

```diff
@@ -40,11 +37,17 @@ def create_export(export_svc_data, org_id, operation_args={}, rbac_filter={}):
         REQUEST_ID_HEADER: exportUUID,
     }
 
+    # x-rh-exports-psk must be an env variable
+    request_headers = {
+        "x-rh-exports-psk": inventory_config.export_service_token,
+        "content-type": HEADER_CONTENT_TYPE["json"] if exportFormat.lower() == "json" else HEADER_CONTENT_TYPE["csv"],
```

Comment:
this can be used like this:

```suggestion
        "content-type": HEADER_CONTENT_TYPE[exportFormat.lower()]
```
As we only receive JSON and CSV types